.github/workflows/dist.yml

```diff
@@ -141,14 +141,14 @@ jobs:
           path: dist
       - name: Create release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_PAT: ${{ secrets.RELEASE_CREATION_TOKEN }}
```

Comment:
How is the PAT created (eg is it's Volkers)? Will it expire?

Reply:
It is mine. No expiration.

Reply:
> (but if we ever worry a leak of PAT, then putting my PAT is less destructive than one of the repo owners.)

Just to check, do you actually have the rights to create a new release? (I.e. can you do this manually?)

Reply:
Yes. I have "admin" role.